### PR TITLE
Zabbix: new module zabbix_user_facts

### PR DESCRIPTION
**Pull request by sky-joker**
##### SUMMARY
This pull request adds a module to get information of Zabbix user.
Tested with Zabbix Server(3.0, 3.2, 3.4, 4.0, 4.2).

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
zabbix_user_facts

### CODE REVIEW

File: lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: zabbix_user_facts
```
Comment (D3DeFi) [inline lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py:16]:
```suggestion
module: zabbix_user_info
```

File: lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+short_description: Gather facts about Zabbix user
```
Comment (D3DeFi) [inline lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py:17]:
```suggestion
short_description: Gather information about Zabbix user
```

File: lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            - Name of the user alias in Zabbix.
+        required: true
+        type: str
+extends_documentation_fragment:
+  - zabbix
+'''
+
+EXAMPLES = '''
+- name: Get zabbix user info
+  zabbix_user_facts:
```
Comment (D3DeFi) [inline lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py:38]:
```suggestion
  zabbix_user_info:
```

File: lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+        "users_status": "0",
+        "usrgrpid": "8"
+      }
+    ]
+  }
+'''
+
+try:
+    from zabbix_api import ZabbixAPI
+    from zabbix_api import Already_Exists
```
Comment (D3DeFi) [inline lib/ansible/modules/monitoring/zabbix/zabbix_user_facts.py:94]: unused import
```suggestion
```

**Review by D3DeFi — CHANGES_REQUESTED** [review]
Good job @sky-joker :) Can you please rename this module to `zabbix_user_info`? There has been currently an initiative (#56822) to fix wrong use of _facts in module names. Part of the documentation that describes it can be found [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_checklist.html#contributing-to-ansible-objective-requirements).

**Comment by sky-joker** [comment]
> There has been currently an initiative (#56822) to fix wrong use of _facts in module names. Part of the documentation that describes it can be found [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_checklist.html#contributing-to-ansible-objective-requirements).

@D3DeFi, Thank you for the review and that infomation!
I will close this plull request once.

**Comment by felixfontein** [comment]
@sky-joker there is no need to close this PR btw, you could simply rename the module in this PR (and adjust the title). Just in case this happens to you again :) (Opening a new PR is of course also OK.)

**Comment by sky-joker** [comment]
@felixfontein 

Thank you for teaching me!!

**Pull request closed without merging**